src/components/header/header.module.scss

```diff
@@ -1,35 +1,41 @@
+@import '@styles/variables.scss';
+
 :root {
     --headerTopBarHeight: 50px;
     --headerNavigationHeight: 54px;
-    --headerHeight: 104px;
+    --headerHeight: calc(var(--headerTopBarHeight) + var(--headerNavigationHeight));
```

Comment:
The reason we didn't define `--headerHeight` as a calc previously is that the Style Panel shows such values as invalid values (with a strike-through).